include/world_builder/utilities.h

```diff
@@ -249,6 +249,21 @@ namespace WorldBuilder
         std::vector<double> m_a, m_b, m_c, m_y;
     };
 
+    struct PointPlaneDistance
```

Comment:
Could you write a bit of documentation above this? Something like: A structure to hold information about the the relationship between a point and a (curved) plane. 

Personally I am not really happy with the name of this struct and the function, since it has grown to more than just a holding distances, but I don't have a better option, so I am open to suggestions, otherwise this will work fine.